packages/@tinacms/graphql/src/build.ts

```diff
@@ -46,9 +46,14 @@ export const indexDB = async ({
     database,
     tinaSchema,
   })
-  const graphQLSchema = await _buildSchema(builder, tinaSchema)
-  // @ts-ignore
-  await database.indexData({ graphQLSchema, tinaSchema })
+  let graphQLSchema: DocumentNode
+  if (database.bridge.supportsBuilding()) {
+    graphQLSchema = await _buildSchema(builder, tinaSchema)
+    await database.putConfigFiles({ graphQLSchema, tinaSchema })
+  } else {
+    graphQLSchema = JSON.parse(await database.bridge.get('_graphql.json'))
```

Comment:
should this include the entire path (`.tina/__generated__/_graphql.json`)?

Reply:
I think so! updated